src/main/java/org/opentripplanner/analyst/SampleSet.java

```diff
@@ -12,12 +12,12 @@
     public final PointSet pset;
 
     /* Vertices at the two ends of a road, one per sample. */
-    Vertex[] v0s;
-    Vertex[] v1s;
+    public Vertex[] v0s;
+    public Vertex[] v1s;
 
     /* Distances to the vertices at the two ends of a road, one per sample. */
-    float[] d0s;
-    float[] d1s;
+    public float[] d0s;
```

Comment:
This would a good place to begin the migration to fixed point math. We could store these in millimeters.